### PR TITLE
Adding the EmailQueue plugin

### PR DESCRIPTION
**Pull request by lorenzo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,6 +130,7 @@ Additional lists you might find useful:
 *Plugins for sending and parsing email.*
 
 - [Gourmet/Email plugin](https://github.com/gourmet/email) - Email helper, layout and more.
+- [EmailQueue plugin](https://github.com/lorenzo/cakephp-email-queue) - Email queue plugin with a preview and sender shell.
```
Comment (ADmad) [inline README.md]: It should be above `Gourmet/Email` in alphabetical order :)

Comment (lorenzo) [inline README.md]: is that a new thing?

Comment (ADmad) [inline README.md]: Nope, awesome list has everything in alphabetical order.

**Comment by bravo-kernel** [comment]
No squash?

**Comment by ADmad** [comment]
@bravo-kernel This is not code, so commit history hardly matters.

**Pull request merged**